development.json

```diff
@@ -80,9 +80,20 @@
     "graphql_urls": [
         "https://graphql.dev.threefold.me/graphql"
     ],
-    "hub_url": "https://hub.threefold.me",
+    "geoip_urls": [
+        "https://geoip.grid.tf/",
+        "https://02.geoip.grid.tf/",
+        "https://03.geoip.grid.tf/",
+        "https://geoip.threefold.me/"
+    ],
+    "hub_urls": [
+        "https://hub.threefold.me"
+    ],
+    "v4_hub_urls": [
+        "https://v4.hub.threefold.me"
+    ],
     "flist_url": "redis://hub.threefold.me:9900",
     "bin_repo": "tf-zos-v3-bins",
     "kyc_url": "https://kyc.devnet.threefold.me",
-    "registrar_url": "https://registrar.prod4.threefold.me"
+    "registrar_url": "https://registrar.dev4.grid.tf"
```

Comment:
can we have this registrar url as an array? is that supported in zos4?

Reply:
nope it only supports one URL. You can open an issue for that if you want